src/drivers/distance_sensor/tfmini/tfmini.cpp

```diff
@@ -895,6 +920,7 @@ tfmini_main(int argc, char *argv[])
 	}
 
 out_error:
-	PX4_ERR("unrecognized command, try 'start', 'test', or 'info'");
+	PX4_ERR("unrecognized command: try 'start', 'test', or 'info'");
+        tfmini::usage();
```

Comment:
@bkueng This adds usage to the case where no value is specified. Is this the right thing to do? Should this _replace_ the use of PX4_ERR?

Reply:
Yes this is good. You can shorten the error message to `PX4_ERR("unrecognized command");`, since the usage shows all possible options.

Reply:
OK done. 

It does feel like the module description and examples are great in docs but "too much" in the console when there is an error like this one - ie you just want to see the usage info at this point.

Reply:
True. On a pixhawk you don't see the description though.